src/Razor/Microsoft.AspNetCore.Razor.Tools/src/GenerateCommand.cs

```diff
@@ -284,19 +301,28 @@ private IReadOnlyList<TagHelperDescriptor> GetTagHelpers(string tagHelperManifes
             }
         }
 
-        private SourceItem[] GetSourceItems(string projectDirectory, List<string> sources, List<string> outputs, List<string> relativePath, List<string> fileKinds)
+        private static SourceItem[] GetSourceItems(List<string> sources, List<string> outputs, List<string> relativePath, List<string> fileKinds, List<string> cssScopeSources, List<string> cssScopeValues)
         {
+            var cssScopeSourceIndices = new Dictionary<string, int>(StringComparer.Ordinal);
```

Comment:
```suggestion
            var cssScopeAssociations = new Dictionary<string, int>(StringComparer.Ordinal);
```

Also, with the `StringComparer.Ordinal` that doesn't always hold true on windows OS`. Is there a [FilePathComparer](https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/FilePathComparer.cs) equivalent available in this MSBuild task?

Reply:
I think it's sufficient to use `StringComparer.OrdinalIgnoreCase` even on Linux in this particular case. That's because we don't support having components whose names differ only by case. Various parts of the system, at compile and runtime (e.g., routing), will not accept that.